src/frontend/scss/colors/_theme.scss

```diff
@@ -348,6 +348,7 @@ $r-theme: (
     plan-title-color: r-color('firebrick6'),
     license-label-color: r-color('denim'),
     checkmark-list-decoration: url('../../richie/images/components/checkmark.svg'),
+    checkmark-list-decoration-color: r-color('indianred3'),
```

Comment:
Do not forget to update the `UPGRADE.md` to mention to add this variable for those who overridden the `course-detail` theme.